### PR TITLE
Feat metrics

### PR DESCRIPTION
**Pull request by nal10**
Outputs cell likelihood metric as a csv

### CODE REVIEW

File: metadata/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+metadata_version: 1
+name: aind-smartspim-classification-test
```
Comment (camilolaiton) [inline metadata/metadata.yml:2]: We should change this name to aind-smartspim-cell-classification

File: code/aind_smartspim_classification/classification.py [hunk 2]
```diff
@@ -151,74 +151,77 @@ def cell_classification(smartspim_config: dict, logger: logging.Logger):
...
-        mask_array, chunks, chunk_step, smartspim_config["mask_scale"] - smartspim_config['downsample']
+        mask_array,
+        chunks,
+        chunk_step,
+        smartspim_config["mask_scale"] - smartspim_config["downsample"],
     )
 
     rechunk_size = [axis * (chunk_step // axis) for axis in signal_array.chunksize]
     signal_array = signal_array.rechunk(tuple(rechunk_size))
+    background_array = background_array.rechunk(tuple(rechunk_size))
```
Comment (camilolaiton) [inline code/aind_smartspim_classification/classification.py:162]: Are we sticking with dask for the classification or we want to use the new package for large-scale prediction?

**Review by camilolaiton — CHANGES_REQUESTED** [review]

File: code/aind_smartspim_classification/classification.py (hunk 2 above)
Comment (nal10) [reply code/aind_smartspim_classification/classification.py:162]: I think we are living right now with dask. I have been testing the new large-scale predictions and have some things we should talk about before switching over but that is the longer term goal

File: metadata/metadata.yml (hunk 1 above)
Comment (nal10) [reply metadata/metadata.yml]: fixed

**Review by camilolaiton — APPROVED** [review]

**Pull request merged**